### DISCUSSION TITLE
Windows support, unable to use arrow keys on changie new

### DISCUSSION THREAD

**Discussion by MCBama** [miniscruff/changie#336] [Q&A] [answered]
When attempting to use changie new the prompt to use arrow keys to select between the labels appears but all arrow presses appear to be ignored and continuing the prompt defaults to "Added" only. Is there a specific configuration I need to setup to enable arrow key inputs with changie? I installed using the manual installation steps as well as using "scoop" as outlined on the changie installation page.

Would it be possible to add additional flags to the changie new command such as `changie new change` to avoid these kinds of issues?

**Comment by miniscruff** [comment, marked as answer]
Can you try using the vim shortcuts of J and K for up and down? 
Also what terminal are you using? 
Changie uses a prompt tool so its kinda outside the scope of work to resolve this, although #119 would probably resolve it.

**Reply by MCBama** [reply]
so when using windows command prompt and power shell j and k work but inside git bash it does not. how interesting...

Well, I can work with that. It's functional enough for me thanks.
